content/en/_index.html

```diff
@@ -72,3 +72,7 @@
   <img src="https://www.netlify.com/img/global/badges/netlify-color-accent.svg" alt="Deploys by Netlify" align="right" />
 </a>
 
+<a href="https://www.redhat.com">
+  <img src="/images/sponsored-by-redhat.png" alt="Sponsored by Red Hat" width="200" height="auto" />
```

Comment:
Can you align this to the right, such that it is grouped with, and left of,  the Netlify image.

Reply:
Well, I tried, but couldn't get it to look right.

Reply:
you tell me how to shift this to the right: https://deploy-preview-724--kiali.netlify.app/